<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>589</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>人靠近</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>拿弓箭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>射 箭 考 试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成报告</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_report">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>测 试 报 告</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>分析打分</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="estlabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="estlabel_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="standard">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>871</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>339</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="est">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>29</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>分 析 评 分 中</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>450</y>
      <width>77</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>报告内容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>测试回顾</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>训练矫正</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>421</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_report_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>你的最终得分是：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_report_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>前手弯曲，扣分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_report_4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>前手不水平，扣分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_report_5">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>后手肘下垂，扣分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_report_6">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>后手腕下垂，扣分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_report_7">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>张弓不充分，扣分：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionDebug_Mode"/>
    <addaction name="actionRelease_Mode"/>
    <addaction name="actionSTART"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDebug_Mode">
   <property name="text">
    <string>Debug Mode</string>
   </property>
  </action>
  <action name="actionRelease_Mode">
   <property name="text">
    <string>Release Mode</string>
   </property>
  </action>
  <action name="actionSTART">
   <property name="text">
    <string>START</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionDebug_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebug_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRelease_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRelease_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebug_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRelease_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebug_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRelease_Mode</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_4</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSTART</sender>
   <signal>triggered()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
